### ISSUE TITLE
Title is overpowering with phones in landscape mode

### ISSUE THREAD

**Issue by Fudge0952** [SvenTiigi/WhatsNewKit#37]
## WhatsNewKit Environment

- WhatsNewKit version: 1.3.1
- Xcode version: 11.3.1 (11C504)
- Swift version: 5
- macOS version running Xcode: 10.15.3 (19D76)
- Dependency manager (SPM, Carthage, CocoaPods, Manually): Manual? I'm using the example project

## What did you do?

* Built & Ran the example project
*  Rotated iPhone to landscape
*  Scrolled through the changelog

## What did you expect to happen?

* I thought perhaps the title would pan with the content so it wasn't so obstructive. Kind of like how "large titles" for navigation controllers work

## What happened instead?

* Title remained the same size and viewing the content was quite difficult.

Perhaps this could be a general improvement to pan the title like how the "large titles" work in navigation controllers? I'm not sure if there was a reasoning behind making it static? I've attached a screenshot of the layout in landscape mode below

![image](https://user-images.githubusercontent.com/3098078/77034682-a7708400-69fe-11ea-8c46-9b4e8fd7137c.png)

**Comment by SvenTiigi** [comment]
Hey @Fudge0952,

Thanks for opening up an issue for WhatsNewKit 🙌

Sadly the layout isn't currently optimized for landscape orientation on iOS.

This is definitely a task on the road map for the next release of WhatsNewKit. 
I was also thinking about implementing a scroll behaviour that is identical to the UINavigationController with large titles.

Please feel free to open up a PR if you have any ideas on how to improve the layout of the `WhatsNewViewController` 👌

**Comment by Fudge0952** [comment]
Hey @SvenTiigi,

I'll definitely do some experimentation using large titles to see if we can benefit from it. I'll keep you updated with what I discover 😉

**Comment by Fudge0952** [comment]
So I was doing some experimentation and in order to support large titles, it would require some reworking of how the views are laid out. Right now the title view, content view, and button view are all separate views, so in order to support shrinking the title when the user scrolls, we would need to modify the layout completely so that the table view and title are together (to avoid nasty hacking with on scroll events). If we wanted to use the built-in "prefers large titles", it would mean bumping the minimum version to iOS 11 (not sure if that's something you're wanting to do. Alternatively, you could include the title as the first cell of the items table view and when it is scrolled out of view, you update the navigation title with the title label's text (and hide it when it comes into focus). This has the potential to limit the customisation of the title layout (unless you still customise it as a child cell of the items table view).

What are your thoughts on all this @SvenTiigi?

**Comment by Fudge0952** [comment]
Hi @SvenTiigi, I had attempted to add an option to allow the title to scroll with the items. The related pull request is [here](https://github.com/SvenTiigi/WhatsNewKit/pull/38). Let me know what you think :)

- Tim

**Comment by SvenTiigi** [comment]
Hi @Fudge0952,

sorry I totally forgot about this issue 🙈 My fault 😃

The PR looks good. Great work 👍 

I will open up a new feature branch to merge your PR 👌

**Comment by SvenTiigi** [comment]
Hey @Fudge0952,

your PR has been merged and is now available within the new release [`1.3.2`](https://github.com/SvenTiigi/WhatsNewKit/releases/tag/1.3.2) of WhatsNewKit 👍 

For a better consistency I've moved the `titleMode` property to the `TitleView` configuration.

### Title Mode
In default the TitleView is sticked to the top. 
If you wish that the TitleView scrolls with the ItemsView you can change the `titleMode` on the TitleView configuration.

```swift
// TitleView scrolls alongside with the ItemsView
configuration.titleView.titleMode = .scrolls

// TitleView is fixed to top
configuration.titleView.titleMode = .fixed
```
> ☝️ In default the titleMode is set to `.fixed`.

**Comment by Fudge0952** [comment]
Excellent, no worries at all. Wasn’t too sure where you wanted the property so I made a guess 😝
